lib/shared/dictionary.dart

```diff
@@ -1,5 +1,5 @@
 const appname= 'My Meals App';
-const addFavoritesMsg= "Start adding some favorites ";
-const yourFilter="Your Filters";
-const yourFavorites="Your Favorites";
-const categories ='Categories';
\ No newline at end of file
+const addFavoritesMsg= 'Start adding some favorites ';
+const yourFilter='Your Filters';
+const yourFavorites='Your Favorites';
+const categories ='Categories';
```

Comment:
@Agushm I askded to have a blank empty line at the end of the file i.e. after line 5. Just press enter after line 5.

Reply:
I have made it like that

Reply:
in last commit